osbs/api.py

```diff
@@ -726,28 +727,54 @@ def import_image(self, name):
 
     @osbsapi
     def get_token(self):
-        return self.os.get_oauth_token()
+        if self.os.use_kerberos:
+            return self.os.get_oauth_token()
+        else:
+            if self.os.token:
+                return self.os.token
+
+            raise OsbsValidationException("no token stored")
 
     @osbsapi
-    def login(self, token):
-        token_file = utils.get_instance_token_file_name(self.os_conf.conf_section)
-        token_file_dir = os.path.dirname(token_file)
+    def login(self, token, username, password):
+        if self.os.use_kerberos:
+             raise OsbsValidationException("can't use login when using kerberos")
+
+        if not token:
+            if username:
+                self.os.username = username
```

Comment:
could simplify this with: `self.os.username = username or raw_input('Username :')`